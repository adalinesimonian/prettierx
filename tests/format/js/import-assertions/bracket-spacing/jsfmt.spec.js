run_spec(__dirname, ["babel"], {
  // [prettierx]: broken-out bracket spacing options
  graphqlCurlySpacing: false,
  exportCurlySpacing: false,
  importCurlySpacing: false,
  objectCurlySpacing: false,
  typeCurlySpacing: false,
  yamlBracketSpacing: false,
  errors: {
    espree: [
      "dynamic-import.js",
      "static-import.js",
      "re-export.js",
      "empty.js",
    ],
    meriyah: [
      "dynamic-import.js",
      "static-import.js",
      "re-export.js",
      "empty.js",
    ],
  },
});
