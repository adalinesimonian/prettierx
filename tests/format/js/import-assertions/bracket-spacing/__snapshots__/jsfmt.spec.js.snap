// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dynamic-import.js - {"graphqlCurlySpacing":false,"exportCurlySpacing":false,"importCurlySpacing":false,"objectCurlySpacing":false,"typeCurlySpacing":false,"yamlBracketSpacing":false} [espree] format 1`] = `
"Unexpected token , (1:20)
> 1 | import(\\"./foo.json\\", { assert: { type: \\"json\\" } });
    |                    ^
  2 |"
`;

exports[`dynamic-import.js - {"graphqlCurlySpacing":false,"exportCurlySpacing":false,"importCurlySpacing":false,"objectCurlySpacing":false,"typeCurlySpacing":false,"yamlBracketSpacing":false} [meriyah] format 1`] = `
"[1:20]: Expected ')' (1:20)
> 1 | import(\\"./foo.json\\", { assert: { type: \\"json\\" } });
    |                    ^
  2 |"
`;

exports[`dynamic-import.js - {"graphqlCurlySpacing":false,"exportCurlySpacing":false,"importCurlySpacing":false,"objectCurlySpacing":false,"typeCurlySpacing":false,"yamlBracketSpacing":false} format 1`] = `
====================================options=====================================
exportCurlySpacing: false
graphqlCurlySpacing: false
importCurlySpacing: false
objectCurlySpacing: false
parsers: ["babel"]
printWidth: 80
typeCurlySpacing: false
yamlBracketSpacing: false
                                                                                | printWidth
=====================================input======================================
import("./foo.json", { assert: { type: "json" } });

=====================================output=====================================
import("./foo.json", {assert: {type: "json"}});

================================================================================
`;

exports[`empty.js - {"graphqlCurlySpacing":false,"exportCurlySpacing":false,"importCurlySpacing":false,"objectCurlySpacing":false,"typeCurlySpacing":false,"yamlBracketSpacing":false} [espree] format 1`] = `
"Unexpected token assert (1:33)
> 1 | export * as bar from \\"bar.json\\" assert { }
    |                                 ^"
`;

exports[`empty.js - {"graphqlCurlySpacing":false,"exportCurlySpacing":false,"importCurlySpacing":false,"objectCurlySpacing":false,"typeCurlySpacing":false,"yamlBracketSpacing":false} [meriyah] format 1`] = `
"[1:38]: Unexpected token: 'identifier' (1:38)
> 1 | export * as bar from \\"bar.json\\" assert { }
    |                                      ^"
`;

exports[`empty.js - {"graphqlCurlySpacing":false,"exportCurlySpacing":false,"importCurlySpacing":false,"objectCurlySpacing":false,"typeCurlySpacing":false,"yamlBracketSpacing":false} format 1`] = `
====================================options=====================================
exportCurlySpacing: false
graphqlCurlySpacing: false
importCurlySpacing: false
objectCurlySpacing: false
parsers: ["babel"]
printWidth: 80
typeCurlySpacing: false
yamlBracketSpacing: false
                                                                                | printWidth
=====================================input======================================
export * as bar from "bar.json" assert { }
=====================================output=====================================
export * as bar from "bar.json";

================================================================================
`;

exports[`re-export.js - {"graphqlCurlySpacing":false,"exportCurlySpacing":false,"importCurlySpacing":false,"objectCurlySpacing":false,"typeCurlySpacing":false,"yamlBracketSpacing":false} [espree] format 1`] = `
"Unexpected token assert (1:33)
> 1 | export { foo2 } from \\"foo.json\\" assert { type: \\"json\\" };
    |                                 ^
  2 |"
`;

exports[`re-export.js - {"graphqlCurlySpacing":false,"exportCurlySpacing":false,"importCurlySpacing":false,"objectCurlySpacing":false,"typeCurlySpacing":false,"yamlBracketSpacing":false} [meriyah] format 1`] = `
"[1:38]: Unexpected token: 'identifier' (1:38)
> 1 | export { foo2 } from \\"foo.json\\" assert { type: \\"json\\" };
    |                                      ^
  2 |"
`;

exports[`re-export.js - {"graphqlCurlySpacing":false,"exportCurlySpacing":false,"importCurlySpacing":false,"objectCurlySpacing":false,"typeCurlySpacing":false,"yamlBracketSpacing":false} format 1`] = `
====================================options=====================================
exportCurlySpacing: false
graphqlCurlySpacing: false
importCurlySpacing: false
objectCurlySpacing: false
parsers: ["babel"]
printWidth: 80
typeCurlySpacing: false
yamlBracketSpacing: false
                                                                                | printWidth
=====================================input======================================
export { foo2 } from "foo.json" assert { type: "json" };

=====================================output=====================================
export {foo2} from "foo.json" assert {type: "json"};

================================================================================
`;

exports[`static-import.js - {"graphqlCurlySpacing":false,"exportCurlySpacing":false,"importCurlySpacing":false,"objectCurlySpacing":false,"typeCurlySpacing":false,"yamlBracketSpacing":false} [espree] format 1`] = `
"Unexpected token assert (1:31)
> 1 | import json from \\"./foo.json\\" assert { type: \\"json\\" };
    |                               ^
  2 |"
`;

exports[`static-import.js - {"graphqlCurlySpacing":false,"exportCurlySpacing":false,"importCurlySpacing":false,"objectCurlySpacing":false,"typeCurlySpacing":false,"yamlBracketSpacing":false} [meriyah] format 1`] = `
"[1:36]: Unexpected token: 'identifier' (1:36)
> 1 | import json from \\"./foo.json\\" assert { type: \\"json\\" };
    |                                    ^
  2 |"
`;

exports[`static-import.js - {"graphqlCurlySpacing":false,"exportCurlySpacing":false,"importCurlySpacing":false,"objectCurlySpacing":false,"typeCurlySpacing":false,"yamlBracketSpacing":false} format 1`] = `
====================================options=====================================
exportCurlySpacing: false
graphqlCurlySpacing: false
importCurlySpacing: false
objectCurlySpacing: false
parsers: ["babel"]
printWidth: 80
typeCurlySpacing: false
yamlBracketSpacing: false
                                                                                | printWidth
=====================================input======================================
import json from "./foo.json" assert { type: "json" };

=====================================output=====================================
import json from "./foo.json" assert {type: "json"};

================================================================================
`;
